music21/converter/__init__.py

```diff
@@ -1048,6 +1048,9 @@ def parse(value: Union[bundles.MetadataEntry, bytes, str, pathlib.Path],
 
     `format` specifies the format to parse the line of text or the file as.
 
+    `compress` specifies whether to immediately compress the .xml output to
+    a .mxl file. The intermediate .xml file will not be retained.
+
```

Comment:
Not sure what compress on parse does for the user -- can you explain?

Reply:
Once I take your suggestion below to get compress into the method signature, then I don't need to expose this here at all. As it was, `compress` was hanging out in `**keywords`, so a user _could_ have passed it in here, setting format '.xml' and then setting compress=True, but there's no reason to support that when `format='mxl'` will do.

Reply:
Haha nope, I definitely cut and pasted this into the wrong method when I was looking for more places to document. Duh.